plugins/module_utils/import_handler.py

```diff
@@ -71,10 +109,36 @@ def method(*args, **kwargs):
 
 class MissingImport(object):
     def __init__(self, import_name=""):
+        """Error when it is unable to import a module due it being missing.
```

Comment:
Just a little typo: instead of `...module due it being missing.`, it should be `...module due to it being missing.`